contributingGuides/PROPOSAL_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Proposal
+
+### Please re-state the problem that we are trying to solve in this issue.
+
+### What is the root cause of that problem?
+
+### What changes do you think we should make in order to solve the problem?
+
+### What alternative solutions did you explore? (Optional)
+
+**Reminder:** Please use plain English, be brief and avoid jargon. Feel free to use images, charts or pseudo-code if necessary. Do not post large multi-line diffs or write walls of text. Do not create PRs unless you have been hired for this job.
+
+<!---
+ATTN: Contributor+
+
+You are the first line of defense in making sure every proposal has a clear and easily understood problem with a "root cause". Do not approve any proposals that lack a satisfying explanation to the first two prompts. It is CRITICALLY important that we at a minimum understand root cause even if the solution doesn't directly address it. When we avoid this step we can end up solving the wrong problems entirely or just duct taping over them.
+
+Instructions for how to review a proposal:
+
+1. Address each contributor proposal one at a time and address each part of the question one at a time e.g. if a solution looks acceptable, but the stated problem is not clear then you should provide feedback and make suggestions to improve each prompt before moving on to the next. Avoid responding to all sections of a proposal at once. Move from one question to the next each time asking the contributor to "Please update your original proposal and tag me again when it's ready for review".
+
+2. Limit excessive conversation and moderate issues to keep them on track. If someone is doing any of the following things please kindly and humbly course-correct them:
+
+- Posting PRs.
+- Posting large multi-line diffs (this is basically a PR).
+- Skipping any of the required questions.
+- Not using the proposal template at all.
+- Suggesting that an existing issue is related to the current issue before a problem or root cause has been established.
+- Excessively wordy explanations.
+
+3. Choose the first proposal that has a reasonable answer to all the required questions.
+-->
```

Comment:
Getting-your-thoughts: Should we also say something that it's OK to hide comments if they are off-topic or no longer need to be considered? I think that can be helpful.

Reply:
Like, maybe if someone doesn't use the proposal template, or they haven't completed all the questions, those are good candidates to be hidden.

Reply:
I like this, but I'm not sure if C+ has moderation tools available.

Reply:
You're right, they do not have that. Do you have any idea if it's easily possible to give them moderation ability? I don't know much about how fine-grained the settings are. CC @johnmlee101 or @coleaeason if you know.

Reply:
sounds perfect!